src/box/box.cc

```diff
@@ -4311,10 +4311,20 @@ box_process_fetch_snapshot(struct iostream *io,
 	}
 
 	say_info("sending current read-view to replica at %s", sio_socketname(io->fd));
```

Comment:
Nit: I'd remove "current" from the log message and say simply "sending read-view to replica at".

Reply:
Removed